property/views.py

```diff
@@ -509,28 +509,23 @@ def get_queryset(self):
             (now() - datetime.timedelta(30)
              ).date(), '%Y-%m-%d'))
         date = dt.strptime(string_date, '%Y-%m-%d')
-        city = self.request.query_params['city']
+
+        city = self.request.query_params.get('city', None)
```

Comment:
I think using `.get()` returns `None` by default if the requested key doesn't exist. No need to pass in `None` explicitly.

Reply:
Your right, I've made the change